pynamodb/expressions/projection.py

```diff
@@ -1,16 +1,20 @@
+from typing import Dict
+from typing import List
+from typing import Union
+
 from pynamodb.attributes import Attribute
 from pynamodb.expressions.operand import Path
 from pynamodb.expressions.util import substitute_names
 
 
-def create_projection_expression(attributes_to_get, placeholders):
+def create_projection_expression(attributes_to_get, placeholders: Dict[str, str]) -> str:
```

Comment:
`attributes_to_get` too?

Reply:
that comes from connection base where it's typed as `Optional[Any]` -- so far I've inferred its:
`Union[List[Attribute], List[Path], List[str], Attribute, Path, str]`
but I was going to punt on it for now